python_ta/transforms/type_inference_visitor.py

```diff
@@ -146,63 +146,46 @@ def visit_name(self, node):
     ##############################################################################
     # Operation nodes
     ##############################################################################
-    def visit_binop(self, node):
-        t1 = self.type_constraints.lookup_concrete(node.left.type_constraints.type)
-        t2 = self.type_constraints.lookup_concrete(node.right.type_constraints.type)
-        op_name = op_to_dunder_binary(node.op)
-
+    def _handle_call(self, node, func_name, *args):
+        """Helper to lookup a function and unify it with given arguments.
+           Returns the return type of unified function call."""
+        arg_types = [self.type_constraints.lookup_concrete(arg) for arg in args]
+        if len(arg_types) == 2:
+            func_call = op_to_dunder_binary(func_name)
+        elif len(arg_types) == 1:
+            func_call = op_to_dunder_unary(func_name)
+        else:
+            raise Exception('Wrong number of arguments given.')
```

Comment:
This isn't correct. `_handle_call` should eventually be able to handle *all* function calls, with an arbitrary number of arguments. The appropriate action in this branch is to just set `func_call = func_name`.